tigerpath/majors_and_certificates/scripts/verifier.py

```diff
@@ -336,8 +336,8 @@ def _init_req_fields(req):
             _init_req_fields(subreq)
     elif "course_list" in req and "excluded_course_list" not in req:
         req["excluded_course_list"] = []
-    elif "num_courses" in req and req["name"] == None and "completed_by_semester" in req:
-        req["name"] = "Complete " + str(req["num_courses"]) + " courses by Semester " + str(req["completed_by_semester"])
+    elif "num_courses" in req:
+        req["min_needed"] = req["num_courses"]
```

Comment:
This, combined with line 538, causes it to report back the number of courses completed out of the total rather than just a yes/no.